Persistence/Contexts/BlogContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Domain.Entities;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace Persistence.Contexts
+{
+    public sealed class BlogContext : IdentityDbContext<ApplicationUser, ApplicationRole, int>
+    {
+        public BlogContext()
+        {
+            
+        }
+
+        public BlogContext(DbContextOptions<BlogContext> options) : base(options)
+        {
+            Database.EnsureCreated();
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer(
+                "Server=sonnov;Database=BlogIdentityDatabase;Trusted_Connection=true;"
```

Comment:
Replace connectionString to appsettings.json
Documentation provided - https://docs.microsoft.com/en-us/ef/core/miscellaneous/connection-strings